docs/src/rules/no-implicit-coercion.md

```diff
@@ -14,6 +14,8 @@ Such as:
 var b = !!foo;
 var b = ~foo.indexOf(".");
 var n = +foo;
+var n = -(-foo);
+var n = foo - 0;
```

Comment:
Can you also add these examples here:

https://github.com/eslint/eslint/blob/a9a17b3f1cb6b6c609bda86a618ac5ff631285d2/docs/src/rules/no-implicit-coercion.md?plain=1#L84-L91